stats/src/test/java/io/airlift/stats/TestTDigest.java

```diff
@@ -257,6 +260,25 @@ public void testMerge()
         assertEquals(merged.valueAt(1), 8.0);
     }
 
+    @Test
+    public void testUnmergeable()
+            throws IOException
+    {
+        // Tests an edge case where centroids cannot be merged and an attempting to add another value fails because of lack of room in the buffer
+        byte[] serialized = Resources.toByteArray(Resources.getResource("io/airlift/stats/unmergeable-tdigest"));
```

Comment:
This can be
```java
getResource(getClass(), "unmergeable-tdigest")

Reply:
I want to be explicit in case this class moves. Also, it's easier to "find" the file by copying and pasting the path. In reality, I could just put it at the top-level, since this is local to the module.